packages/experiments/src/components/Chiclet/Chiclet.base.tsx

```diff
@@ -32,14 +32,16 @@ export class ChicletBase extends BaseComponent<IChicletProps, IChicletState> {
       case ChicletSize.medium:
```

Comment:
Since we aren't relying on the state anymore, let's remove the following lines from this render function:

```tsx
const { size, footer, description } = this.props;
const { chicletCardProps } = this.state;
```

Then we can just pass {...this.props} into the ChicletCard and ChicletXsmall below, since footer and description are coming from this.props!

```tsx
return <ChicletCard {...this.props} onClick={this._onClick} />;
```
```tsx
return <ChicletXsmall {...this.props} onClick={this._onClick} />;
```